### PR TITLE
Add attestation

### PR DESCRIPTION
**Pull request by huoqifeng**
Signed-off-by: huoqifeng <huoqif@cn.ibm.com>

### CODE REVIEW

File: 安全容器/虚拟机的认证.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
+# 认证(Attestation)
+
+Attestation 就是要验证应用所在的software和hatdware stack是安全和可信的。
```
Comment (wyuany) [inline 安全容器/虚拟机的认证.md:3]: hardware?

File: 安全容器/虚拟机的认证.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+1. 就是拉取容器镜像的环境是安全的，是一个 Trusted Execution Environment (TEE)。
+2. 虚拟机得到了验证，保存的用于验证容器签名的密钥是没有被篡改过的。
+
+所以这是一个信任的链条，当虚拟机的认证没办法保证的时候，容器的认证也就无从谈起了。
+
+## 如何验证虚拟机镜像
+
+### AWS Nitro
+
+[aws-nitro-enclaves](https://github.com/aws/aws-nitro-enclaves-nsm-api/blob/main/docs/attestation_process.md) 介绍了 Nitro的一些细节
```
Comment (wyuany) [inline 安全容器/虚拟机的认证.md:39]: github.com的链接在微信打不开吧？

File: 安全容器/虚拟机的认证.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+
+### IBM LinuxONE Secure Execution
+
+[LinuxONE](https://www.ibm.com/docs/en/linux-on-systems?topic=tasks-secure-guest) 介绍了 IBM Secure Execution的一些细节
+
+## 如何验证容器镜像
+
+前面提到了，容器镜像的安全是以虚拟机的安全为前提的，Kata就是一个这样的容器运行时环境。
```
Comment (wyuany) [inline 安全容器/虚拟机的认证.md:47]: 这个二级标题##，内容是不是有点少？

File: 安全容器/虚拟机的认证.md (hunk 1 above)
Comment (huoqifeng) [reply 安全容器/虚拟机的认证.md]: Fixed

File: 安全容器/虚拟机的认证.md (hunk 3 above)
Comment (huoqifeng) [reply 安全容器/虚拟机的认证.md:71]: Added more contents

**Comment by huoqifeng** [comment]
@wyuany review it again?

**Comment by wyuany** [comment]
LGTM

**Review by wyuany — APPROVED** [review]
LGTM

**Pull request merged**